### DISCUSSION TITLE
Load from http get?

### DISCUSSION THREAD

**Discussion by TravisLRiffle** [pjlamb12/runtime-config-loader#31] [General]
I was trying to find an example of how you'd load this from an HTTP get.  What should my API be returning back?   
I want to pull back values for MSAL values which is configured at the @NGModule level.  I *think* this library will do that but I'm not entirely sure.  Thanks for your response.

**Comment by pjlamb12** [comment]
Hey @TravisLRiffle, I'm so sorry for taking so long to get back to you. I never got notified about a new discussion being created.

You should just pass the endpoint for the configuration as the configUrl property:

```
RuntimeConfigLoaderModule.forRoot({ configUrl: 'https://myconfig.com/config' })
```

And that should work the same as passing the path to a JSON file.

It should come back as JSON for now, although it would be nice to add the ability to manipulate the data that comes back.
